packages/router5/index.d.ts

```diff
@@ -130,14 +138,16 @@ declare module 'router5/create-router' {
     }
 
     export interface Options {
-        defaultRoute: string
-        defaultParams: Params
-        trailingSlash: boolean
-        useTrailingSlash: boolean
-        autoCleanUp: boolean
-        strictQueryParams: boolean
-        allowNotFound: boolean
-        strongMatching: boolean
+        defaultRoute?: string
+        defaultParams?: Params
+        strictTrailingSlash?: boolean
+        trailingSlashMode?: TrailingSlashMode
+        queryParamsMode?: QueryParamsMode
+        autoCleanUp?: boolean
+        allowNotFound?: boolean
+        strongMatching?: boolean
+        rewritePathOnMatch?: boolean
+        queryParams?: QueryParamsOptions
```

Comment:
Why are all these properties optional? Most of them are guaranteed non-null, given there is [a set of defaults](https://github.com/router5/router5/blob/df70f37236073505a1f619799c9ef2d5b31a8e23/packages/router5/modules/create-router.js#L11-L19).

Reply:
You're right, I thought it was the type definition passed to `createRouter`, not the one returned by `getOptions()`. Correcting it